src/com/android/calendar/event/EditEventView.java

```diff
@@ -377,28 +373,7 @@ public void onNothingSelected(AdapterView<?> arg0) {
         if (tzpd != null) {
             tzpd.setOnTimeZoneSetListener(this);
         }
-        TimePickerDialog tpd = (TimePickerDialog) fm.findFragmentByTag(FRAG_TAG_TIME_PICKER);
-        if (tpd != null) {
-            View v;
-            mTimeSelectedWasStartTime = timeSelectedWasStartTime;
```

Comment:
Why are you dropping this? (Haven't looked what it is used for exactly but timeSelectedWasStartTime Is a constructor parameter.)